cli/src/plugins/javadoc/index.ts

```diff
@@ -46,6 +49,28 @@ function getClassType(doc: ParsedClassDoc): ClassType {
   }
 }
 
+function getType(doc: ParsedClassDoc): EntityType {
+  if (doc.isError) {
+    return "error";
+  } else if (doc.isException) {
+    return "exception";
+  } else if (doc.isEnum) {
+    return "enum";
+  } else if (doc.isInterface) {
+    return "interface";
+  } else {
+    return "class";
+  }
+}
+
+function capitalize(str: string): string {
+  if (str == null || str.length < 1) {
```

Comment:
How can str be null? Is the type correct in the parameter list?

Reply:
ah that's fair, i was just being overly Java-y safe here in case someone passes a null string. I forget that TS prevents that from happening!

Reply:
TS doesn't actually prevent it from happening! For example, the value could come from a JSON file and we just tell TS that it's a string, but we forgot to validate somewhere. Passing it in would then be fine. However, that would indicate a bug elsewhere.